bigtable-dataflow-parent/bigtable-beam-import/src/test/resources/log4j.properties

```diff
@@ -22,3 +22,8 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
 log4j.category.org.apache.beam.sdk.io.FileBasedSource=WARN
 log4j.category.com.google.cloud.bigtable.beam.sequencefiles.SequenceFileSource=WARN
+# make hbase snapshot import integration tests output less verbose.
+log4j.category.org.apache.hadoop.hbase.snapshot.RestoreSnapshotHelper=WARN
```

Comment:
can the entirety of org.apache.hadoop be set to WARN?

Reply:
Done